src/LSST/control/Model.cpp

```diff
@@ -25,20 +25,42 @@
 // System headers
 
 // Project headers
+#include "control/PowerSystem.h"
 #include "control/State.h"
+#include "util/Log.h"
 
 namespace LSST {
 namespace m2cellcpp {
 namespace control {
 
-Model::Model() {}
+Model::Model() {
+    LDEBUG("Model::Model() creating PowerSystem");
+    _powerSystem = PowerSystem::Ptr(new PowerSystem());
+}
 
 State::Ptr Model::getCurrentState() { return _stateMap.getCurrentState(); }
 
 std::shared_ptr<State> Model::getState(std::string const& stateName) { return _stateMap.getState(stateName); }
 
 bool Model::changeState(std::shared_ptr<State> const& newState) { return _stateMap.changeState(newState); }
 
+bool Model::goToSafeMode(std::string const& note) {
+    if (_powerSystem == nullptr) {
+        LERROR("Model::goToSafeMode _powerSystem is nullptr");
+        return false;
+    }
+    // TODO: DM-40339 Add code to put the system in safe mode,
+    //   If the system is already in safe mode, return false

```

Comment:
Is the format correct between the `//` and `If`? It does not align well with the above line 52.